src/main/app/src/components/hakutulos/MobileToggleFiltersButton.tsx

```diff
@@ -32,6 +34,14 @@ const useStyles = makeStyles(() => ({
     transform: 'translateX(-50%)',
     backgroundColor: colors.brandGreen,
   },
+  fixedMenuOpen: {
+    position: 'fixed',
+    zIndex: 1,
+    bottom: 30,
+    left: '65%',
```

Comment:
Voit passata `useStyles`-funktiolle `{menuOpen}` ja sitten tehdä tässä näin. Ei tarvi kopioida noita muita tyylejä.
```suggestion
    left: ({menuOpen}) => menuOpen ? '65%' : '50%',
```

Reply:
Annetaan tosiaan tämän nyt olla, kun ei onnistunutkaan ihan näin simppelisti, eikä ole kovin olennainen parannus.